main.go

```diff
@@ -101,13 +101,16 @@ func run() int {
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
-	err = cmd.Run()
 
+	err = cmd.Run()
 	code := cmd.ProcessState.ExitCode()
-	if code == 0 {
-		println(string(bk))
+	if code == 0 && err == nil {
+		if verbose {
+			println("[done]", string(bk))
```

Comment:
Do not use println. pleaes use fmt.Fprint or fmt.Fprintln or fmt.Fprintf instead.

Reply:
ok, i felt unfamiliar with this builtin 'println' also, but it was there, so..